libs/design-system/src/atoms/icons/svg/ArrowPath.tsx

```diff
@@ -1,11 +1,12 @@
 import { SVGIconProps } from '.';
+import { cn } from '../../..';
 
 export const ArrowPath = ({ className, size }: SVGIconProps) => (
   <svg
     xmlns="http://www.w3.org/2000/svg"
     width={size}
     height={size}
-    className={className}
+    className={cn('plasmo-shrink-0', className)}
```

Comment:
Not needed. I'll explain in another comment why 😉
Same for all the other ones

Reply:
Tell me if I forgot something but:
1. I've committed your suggestion below (as you can see)
2. Pulled the code in my local branch
3. Fooled around with the `plasmo-shrink-0` class

Here's what happened:
![image](https://github.com/Bastien-and-Gauvain/monorepo/assets/45038783/fa85f2e0-de56-49aa-8f15-7314aa0be471)
![image](https://github.com/Bastien-and-Gauvain/monorepo/assets/45038783/415d0145-7ce5-4117-a9a0-c918a29e8b3e)

Therefore (except if I'm missing smthg), we need `plasmo-shrink-0`

Reply:
You know what? Let's forget about it. I suggest merging the PR as is, with my plasmo-shrinks everywhere, because, in the end, all will be replaced by the icons from `heroicons`. And as you can see, no unintended behavior with them:
![image](https://github.com/Bastien-and-Gauvain/monorepo/assets/45038783/b5f45166-7dd5-4953-a58f-8907abaa4449)

**First one who's blocked by this pending PR can squash-merge it, as far as I'm concerned!**